docs/troubleshooting.md

```diff
@@ -132,7 +139,7 @@ Users can view existing package names with the following command:
 $ oc get packagemanifests -n openshift-marketplace
 ```
 
-## Where to go for help
+# Where to go for help
 
 * #forum-operators channel in the [upstream Kubernetes slack instance](https://slack.k8s.io/)
```

Comment:
This is the wrong channel name. It is [kubernetes-operators](https://kubernetes.slack.com/messages/CAW0GV7A5). Please fix as part of this PR.

Reply:
@aravindhp addressed.